types/react-native/index.d.ts

```diff
@@ -162,6 +162,31 @@ interface EmitterSubscription extends EventSubscription {
     remove(): void;
 }
 
+interface DEPRECATED_RCTExport<T> {
+	getConstants(): Function;
```

Comment:
[The upstream Flow type](https://github.com/facebook/react-native/blob/v0.61.5/Libraries/TurboModule/RCTExport.js#L33) states that this property should be optional and is a method that returns an object:

```suggestion
	getConstants?: () => {};
```